content_server/openttd/protocol/enums.py

```diff
@@ -0,0 +1,29 @@
+import enum
+
+
+# Copy from OpenTTD/src/network/core/tcp_content.h
+class PacketTCPContentType(enum.IntEnum):
+    PACKET_CONTENT_CLIENT_INFO_LIST = 0  # Queries the content server for a list of info of a given content type
+    PACKET_CONTENT_CLIENT_INFO_ID = 1  # Queries the content server for information about a list of internal IDs
+    PACKET_CONTENT_CLIENT_INFO_EXTID = 2  # Queries the content server for information about a list of external IDs
+    PACKET_CONTENT_CLIENT_INFO_EXTID_MD5 = (
+        3  # Queries the content server for information about a list of external IDs and MD5
```

Comment:
can you shorten the comment to avoid this ugly line wrapping, i wonder?

Reply:
Sadly, this is how the OpenTTD client names it. I kinda don't want to break that identification here. So meh .. line-wrapping it is.